src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs

```diff
@@ -269,8 +269,22 @@ private async Task<List<CommPkg>> GetCommPkgScopeAsync(IList<string> commPkgNos,
                     
                     if (fr.UsePersonalEmail != null && fr.UsePersonalEmail == false && fr.Email != null)
                     {
-                        participants.Add(new BuilderParticipant(ParticipantType.Required,
-                            new ParticipantIdentifier(fr.Email)));
+
```

Comment:
I would like to see update tests for your changes

Reply:
I guess you mean Unit tests @paal123 .. and I agree